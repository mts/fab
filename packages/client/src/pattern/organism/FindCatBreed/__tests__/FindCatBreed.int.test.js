import { defaultRender } from './FindCatBreed.int.render'

jest.mock('../../../../store/thunk/completeState', () => ({ setAppCompleteStateThunk: () => {} }))

describe('<FindCatBreed />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
