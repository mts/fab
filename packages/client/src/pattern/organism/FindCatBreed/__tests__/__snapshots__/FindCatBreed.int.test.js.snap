// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FindCatBreed /> Snaphot must match defaultRender 1`] = `
<div
  className="breed"
>
  <h2>
    A random image of 
    beng
     breed
  </h2>
  <h4>
    Type in new breed then press enter or click search button
  </h4>
  <div
    className="breedSearch"
  >
    <input
      className="input"
      onKeyUp={[Function]}
      placeholder="beng"
    />
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Search Cat
    </button>
  </div>
  <div
    className="breedShow"
  >
    <div
      className="image"
    >
      <img
        alt="beng"
        src="https://66.media.tumblr.com/tumblr_m4iejdP8Hb1qd477zo1_250.jpg"
      />
    </div>
  </div>
</div>
`;
